lib/mongo/auth/aws/request.rb

```diff
@@ -86,7 +86,7 @@ def initialize(access_key_id:, secret_access_key:, session_token: nil,
         # @return [ String ] formatted_time ISO8601-formatted time of the
         #   request, as would be used in X-Amz-Date header.
         def formatted_time
-          @formatted_time ||= @time.utc.strftime('%Y%m%dT%H%M%SZ')
+          @formatted_time ||= @time.getutc.strftime('%Y%m%dT%H%M%SZ')
```

Comment:
Not sure where this came from! Seems to be something I did 2 summers ago that never got merged.